Add await check to make DuckDuckGo test succeed in Firefox

gabelilly opened the pull request:
I was trying to get into using the tool and found that the default example test failed when using the Firefox driver option.

This PR adds a line to the DuckDuckGo examples which waits for the main home page search bar to not be on the page before checking the title. I did not alter the Spock test since I'm not very familiar with the framework. My editor also removed trailing spaces.

Please suggest better solutions to the failure. Thanks very much.

filipcynarski commented on examples/quickstart/src/test/java/org/fluentlenium/examples/hooks/ExampleHookTest.java:
```diff
@@ -14,6 +16,7 @@ public void titleOfDuckDuckGoShouldContainSearchQueryName() {
         goTo("https://duckduckgo.com");
         $("#search_form_input_homepage").fill().with("FluentLenium");
         $("#search_button_homepage").submit();
+        await().atMost(5, TimeUnit.SECONDS).until($("#search_form_homepage")).not().present();
```
use el insted of $ as far as you don't need to operate on list

gabelilly commented:
Hey @filipcynarski,

Changed those over. The Travis build fails, but I think that's before the line I changed. Let me know if I need to do anything else on this. Thanks.

filipcynarski commented:
No worries! Thank you @gabelilly
